lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2018,6 +2025,12 @@ definitions:
         type: boolean
         example: true
         default: true
+      subscriberdb_sync_interval:
```

Comment:
Why two `subscriberdb_sync_interval` ? Create a separate object definition and then reference it in each location